ts/data.ts

```diff
@@ -1 +1,23 @@
-/* exported data */
+const API_KEY = 'd96ee95088b9438aa8c07a785c81b6e9';
+const API_URL = 'https://api.sportsdata.io/v4/soccer/scores/json/Teams/EPL';
+
+export interface Team {
+  TeamId: number;
+  Name: string;
+  WikipediaLogoUrl: string;
+}
+
+const TEAM_NAMES = [
+  'Manchester United',
+  'FC Barcelona',
+  'Real Madrid',
+  'Arsenal',
+  'PSG',
+  'Liverpool',
+];
+
+export async function fetchTeams(): Promise<Team[]> {
+  const response = await fetch(`${API_URL}?key=${API_KEY}`);
+  const data = await response.json();
+  return data.filter((team: Team) => TEAM_NAMES.includes(team.Name)) as Team[];
+}
```

Comment:
You are using `async/await` here but you don't have a `try/catch` block.  How are you going to handle if you get any errors? Also you should can handle your render here